homeassistant/components/vicare/strings.json

```diff
@@ -242,6 +242,44 @@
       "volumetric_flow": {
         "name": "Volumetric flow"
       },
+      "ess_state_of_charge": {
+        "name": "Battery"
+      },
+      "ess_power_current": {
+        "name": "Battery power"
+      },
+      "ess_state": {
+        "name": "Battery state",
+        "state": {
+          "charge": "Charge",
+          "discharge": "Discharge",
+          "standby": "Standby"
+        }
+      },
+      "pcc_current_power_exchange": {
+        "name": "PCC power exchange"
+      },
+      "pcc_energy_consumption": {
+        "name": "PCC energy consumption"
+      },
+      "pcc_energy_feed_in": {
+        "name": "PCC energy feed-in"
+      },
+      "photovoltaic_power_production_current": {
+        "name": "Solar power"
+      },
+      "photovoltaic_energy_production_today": {
+        "name": "Solar energy production today"
+      },
+      "photovoltaic_status": {
+        "name": "Solar state",
+        "state": {
+          "nothing": "Nothing",
+          "ready": "Ready",
+          "production": "Production",
+          "unknown": "Unknown"
```

Comment:
If the value is Unknown, we should return `None` as sensor value instead

Reply:
Just like this? 
```suggestion
          "unknown": None
```

Reply:
No I mean in the code, so currently we return "unknown" when its unknown (the native value of the entity), but it should return None

Reply:
Can this be done in the lambda?

Reply:
it could be a "return if ... is "unknown" None else ..." thing. But if it spans more than 1 line it should be a separate function

Reply:
see https://github.com/home-assistant/core/pull/106600/commits/cdaf6c695be760d1b23763a2230e73c623d11a08

Reply:
It would be maybe nicer to put in a separate function